runtime/Stdlib_Float.resi

```diff
@@ -37,7 +37,7 @@ Float constants.
 module Constants: {
   /**
   The special value "Not a Number"
-  See [`NaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) on MDN.
+See [`NaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) on MDN.
```

Comment:
Not sure why this whitespace is removed.

Reply:
that looks like a bug of the formatter when there's some indentation and the line is too long I guess?